packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -253,6 +253,38 @@ Variables used for function parameters must match the function data type. For ex
 | escapeHtml(String html)                                       | Escapes the characters in the provided string using HTML entities                                                             |
 | escapeHtmlAttr(String html)                                   | Encodes data for use in HTML attributes                                                                           |
 
+## Customize a forgotten password recovery email
+
+Okta can send an email notification to an end user who has forgotten their password. They must follow the provided link to complete the forgot password process.
+
+If you are using an [embedded authentication](/docs/concepts/redirect-vs-embedded/) deployment model for your application's sign-in flow, you can create a customized email magic link for your **Forgot Password** email template.
```

Comment:
why is embedded model important here? is deployment model relevant in terms of customizing this email template?

Reply:
@albertchen-okta According to @zeusandrade-okta :
> admins can customize the templates as desired
> if they want to have a custom magic link in the template, it should be for embedded

Reply:
is he saying redirect customers CANNOT use custom magic link? makes no sense but im not close to the feature